torchsparse/nn/modules/conv.py

```diff
@@ -143,53 +230,65 @@ def forward(self, inputs):
                             flatten.indices().t().int(), ratio)
 
 
-class ToDenseBEVConvolution(nn.Module):
+class ToBEVHeightCompression(nn.Module):
+    """
+    
+    Converts a torchsparse.SparseTensor to a dense volumetric tensor,
+    then flatten the z dimension.
+    E.g. input [N, C] (assume batch_size=1), spatial size [128,2,128]
+    then output will be [1, 2C, 128, 128]
+    
+    channels: input channels 
+    (Note: output channels = channels x #unique z values)
+    shape: shape of BEV map.
+    dim: dimension index for z. (default: 1 for KITTI coords)
+    bias: whether to use bias.
+    
+    
+    """
     def __init__(self,
-                 in_channels: int,
-                 out_channels: int,
+                 channels: int,
                  shape,
```

Comment:
What would be the type for `shape`? From the code below, it seems to be `torch.Tensor`. It would be better to support `Tuple` or `List` as it would be much easier to use.

Reply:
Fixed.